omod/src/main/webapp/resources/htmlforms/ovc/ovcDiscontinuation.html

```diff
@@ -17,6 +17,24 @@
 		var enrollmentDate = ovcDiscontinuation_VELOCITY.split(",")[0].split(":")[1];
```

Comment:
Let's consider using objects with keys and values instead of a string that requires splitting for such.